test/e2e/arktos/multi_tenancy/test_suites/kubectl/test_describe.yaml

```diff
@@ -0,0 +1,79 @@
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Kubectl describe Tests ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This test suite verifies command "kubectl describe".
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+######################################################
+# test variables
+######################################################
+Variables:
+  test_tenant: random_8
+  test_ns: random_8
+
+###########################################################################################################
+# test setup
+###########################################################################################################
+Tests:
+  - BeforeTestMessage: Starting test setup ...
+    Command: ${kubectl} create tenant ${test_tenant}
+    OutputShouldContain: 
+    - "\ntenant/${test_tenant} created\n"
+
+  - Command: ${kubectl} create namespace ${test_ns} --tenant ${test_tenant}
+    OutputShouldBe: "namespace/${test_ns} created\n"
+
+  - Command:  ${kubectl} apply -f ${test_data_dir}/sample-deployment.yaml -n ${test_ns} --tenant ${test_tenant} 
+    OutputShouldBe: "deployment.apps/sample-nginx-deployment created\n"
+
+############################################
+# test "kubectl describe"
+############################################
+  - BeforeTestMessage: "Verifying \"kubectl describe\" ..."
+    Command:  ${kubectl} describe tenant ${test_tenant} | grep -E 'Name:|Namespace:|Tenant:' | tr -d ' '
+    OutputShouldContain: 
+    - "Name:${test_tenant}\n"
```

Comment:
also here, do we need these \n at the end for matching?

Reply:
"\n" is not required in this case. However, it keeps the test more accurate. So if the output is mistaken to be  "Name:${test_tenant}a\n", the bug will be caught if the test string is "Name:${test_tenant}\n", but not if it is "Name:${test_tenant}".